sections/slideshow.liquid

```diff
@@ -114,7 +114,7 @@
           {%- endif -%}
         </div>
         <div class="slideshow__text-wrapper banner__content banner__content--{{ block.settings.box_align }} page-width{% if block.settings.show_text_box == false %} banner--desktop-transparent{% endif %}">
-          <div class="slideshow__text banner__box content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
+          <div class="slideshow__text banner__box {% if block.settings.heading != blank or block.settings.subheading != blank or block.settings.button_label != blank %}banner__box-padding {% endif %}content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
```

Comment:
For the ease of reading we tend to put the conditional statements for classes at the end. So that when reading you can see first the ones that are applied at all times, and then the ones that based on a condition.
```suggestion
          <div class="slideshow__text banner__box content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}{% if block.settings.heading != blank or block.settings.subheading != blank or block.settings.button_label != blank %} banner__box-padding{% endif %}">
```